po/de.po

```diff
@@ -238,27 +236,27 @@ msgstr "Seiten teilen"
 
 #: data/menu.ui.h:15
 msgid "Cu_t"
-msgstr ""
+msgstr "Ausschneiden"
 
 #: data/menu.ui.h:16
 msgid "_Copy"
-msgstr ""
+msgstr "Kopieren"
 
 #: data/menu.ui.h:17
 msgid "Paste _After"
-msgstr ""
+msgstr "Einfügen nach Auswahl"
 
 #: data/menu.ui.h:18
 msgid "Paste _Before"
-msgstr ""
+msgstr "Einfügen vor Auswahl"
 
 #: data/menu.ui.h:19
 msgid "Paste Interleave _Odd"
-msgstr ""
+msgstr "Verzahnt einfügen ungerade"
 
 #: data/menu.ui.h:20
 msgid "Paste Interleave _Even"
-msgstr ""
+msgstr "Verzahnt einfügen gerade"
```

Comment:
@timoschwarzer These are the best translations I could come up with, maybe you have got better ideas for them? I am native speaker but I do 99% of computer related stuff in english (not counting [heise.de news](https://heise.de) of course :wink: )